source/usage-examples/changeStream.txt

```diff
@@ -7,9 +7,12 @@ Watch for Changes
 Open a Change Stream
 --------------------
 
-You can watch for changes to a single collection, a database, or an entire deployment in MongoDB with **Change Streams**.
-Open a change stream by calling the ``watch()`` method on a ``Collection``, ``Db``, or ``MongoClient`` object. The
-change stream emits **change event** documents when they occur.
+You can watch for changes in MongoDB using the 
+`Collection.watch() <{+api+}/classes/Collection.html#watch>`__, 
+`Db.watch() <{+api+}/classes/Db.html#watch>`__, and 
+`MongoClient.watch() <{+api+}/classes/MongoClient.html#watch>`__
+methods. For each object, the ``watch()`` method opens a 
+**change stream** to emit **change event** documents when they occur.
```

Comment:
issue: I think this is good, but it presents a bit messy in the page IMO. What are your thoughts about something like

You can watch for changes in MongoDB using the ``watch()`` method on a:
• Collection <link>
• Database <link>
• Entire deployment <link>


And, also keep the API links at the bottom in case someone skims past these. It doesn't hurt to have them in both places.